src/super_gradients/__init__.py

```diff
@@ -1,8 +1,8 @@
 from super_gradients.training import ARCHITECTURES, losses, utils, datasets_utils, DataAugmentation, \
-    TestDatasetInterface, SegmentationTestDatasetInterface, DetectionTestDatasetInterface, ClassificationTestDatasetInterface, SgModel
+    TestDatasetInterface, SegmentationTestDatasetInterface, DetectionTestDatasetInterface, ClassificationTestDatasetInterface, SgModel, KDModel
 from super_gradients.common import init_trainer, is_distributed
 from super_gradients.examples.train_from_recipe_example import train_from_recipe
```

Comment:
Please also add an import for train_from_kd_recipe_example so users could run it from the command line without all the python path overhead.

Reply:
Added in 5688491a77aff88cbc04113e6b79d54a8c88d8d2